SU2_CFD/include/solver_structure.hpp

```diff
@@ -5204,6 +5258,10 @@ class CNSSolver : public CEulerSolver {
   AllBound_MaxHF_Visc; /*!< \brief Maximum heat flux (viscous contribution) for all boundaries. */
   su2double StrainMag_Max, Omega_Max; /*!< \brief Maximum Strain Rate magnitude and Omega. */
   
+  /* Sliding meshes variables */
+
+  su2double ***SlidingState;
+  
```

Comment:
Hi Giulio @LaSerpe one very last thing. I think this pointer is inherited from CEuler, so it should't be redefined.